README.md

```diff
@@ -35,6 +35,44 @@ or, if you want to start the development environment
 npm run dev
 ```
 
+## Running [rclone](https://rclone.org) against Permanent.org instances
+
+- Install rclone, using [official install guide](https://rclone.org/install/)
+
+### Setting up an rclone remote
+
+- `rclone config`
+- Select option n for new and enter a remote name such as `permanent-prod` or `permanent-dev`.
+- Set `SFTP` as storage protocol by by selecting from displayed menu, after name prompt.
+- Set an `ssh` host, the host format is `sftp.${environment}.permanent.org` for example `sftp.dev.permanent.org` for `dev` and `staging`. The prod environment is an exception and simply has host as `sftp.permanent.org`.
```

Comment:
Can you wrap this line? Also is it really `sftp.dev.permanent.org` for both dev and staging?

Reply:
Thanks @cecilia-donnelly, I would rephrase that... What I really mean is that, the `environment` part of the `ssh` host is dynamic for `dev` and `staging`.

So the user, can either use `sftp.dev.permanent.org` for dev or `sftp.staging.permanent.org`...

What about 

> Set an `ssh` host, the host format is `sftp.${environment}.permanent.org` where `${environment}` is either `dev` or `staging` for the dev and staging environments respectively.  For example `sftp.dev.permanent.org.  Prod simply has ssh host as `sftp.permanent.org`.